README.md

```diff
@@ -79,6 +79,25 @@ runs faster.
 
 Reads the Puppetfile in the current directory and installs them under the `path` provided as an argument.
 
+### r10k:validate[path]
+
+Validate the git urls and branches, refs, or tags. Outputs a nice table.
```

Comment:
what exactly does it validate, the syntax of the urls?

Reply:
I'll update the readme, but it validates the url by checking to make sure it actually works and can be cloned.  Then it validates the specific tag, branch or ref exists in the remote repository specified by the url. 

A syntax check would only verify the Puppetfile is valid ruby code only.  If a user supplies incorrect refs and urls this validate command would check that.